spec/radix_tree_spec.rb

```
@@ -259,100 +209,51 @@
     it 'evicts old cache entries when cache gets large' do
       route = double('route')
       tree.add('/path/:id', ['GET'], route)
-
-      # Add many different paths to trigger cache eviction
-      (1..3000).each do |i|
-        tree.find("/path/#{i}", 'GET')
-      end
-
-      cache = tree.instance_variable_get(:@split_cache)
+      (1..3000).each { |i| tree.find("/path/#{i}", 'GET') }
+      cache     = tree.instance_variable_get(:@split_cache)
```

RuboCop (1.76.1)

[warning] Useless assignment to variable - `cache`. Did you mean `cache_max`?

(Lint/UselessAssignment)